cli/src/status.rs

```diff
@@ -31,7 +31,7 @@ impl Status {
         } else {
```

Comment:
Nit:

Your commit subject should adhere to the following standard suggested [here](https://github.com/bottlerocket-os/bottlerocket/pull/2219#issuecomment-1176887596):

>https://cbea.ms/git-commit/
 Minor changes:
>- don't capitalize the subject
>- start with a "subsystem: blah" tag unless the change is very cross-cutting